src/wiki/keyboard-shortcuts.md

```diff
@@ -0,0 +1,208 @@
+| <kbd>Ctrl</kbd> + <kbd>W</kbd>                    | Closes the active window.                                                       |
+| <kbd>Ctrl</kbd> + Mouse scroll wheel              | Adjusts the dimensions and look of file and folder icons.                       |
+| <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>E</kbd> | Shows all folders located above the chosen folder.                              |
+| <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>N</kbd> | Creates a new folder.                                                           |
+| <kbd>Num Lock</kbd> + <kbd>\*</kbd>               | Shows all subfolders within the chosen folder.                                  |
+| <kbd>Num Lock</kbd> + <kbd>-</kbd>                | Shows the items within the chosen folder.                                       |
+| <kbd>Alt</kbd> + <kbd>P</kbd>                     | Shows the Preview panel.                                                        |
+| <kbd>Alt</kbd> + <kbd>Enter</kbd>                 | Opens the Properties window for the chosen item.                                |
+| <kbd>Alt</kbd> + <kbd>→</kbd>                     | Views the next folder.                                                          |
+| <kbd>Alt</kbd> + <kbd>↑</kbd>                     | Navigates to the parent folder of the current directory.                        |
+| <kbd>Alt</kbd> + <kbd>←</kbd>                     | Views the previous folder.                                                      |
+| <kbd>Backspace</kbd>                              | Views the previous folder.                                                      |
+| <kbd>→</kbd>                                      | Expand the current selection (if collapsed), or highlight the first subfolder.  |
+| <kbd>←</kbd>                                      | Collapse the current selection (if expanded), or navigate to the parent folder. |
+| <kbd>End</kbd>                                    | Views the bottom of the active window.                                          |
+| <kbd>Home</kbd>                                   | Views the top of the active window                                              |
+| <kbd>F11</kbd>                                    | Enters full screen mode.                                                        |
+
+# Taskbar Keyboard Shortcuts
+
+| Combinations                                                | Actions                                     |
+| ----------------------------------------------------------- | ------------------------------------------- |
+| <kbd>Shift</kbd> + Click a taskbar button                   | Opens an app or another instance of an app. |
+| <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + Click a taskbar button | Opens the app as administrator.             |
+| <kbd>Shift</kbd> + Right click a taskbar button             | Shows the context menu for the app.         |
+
+# Virtual desktops keyboard shortcuts
```

Comment:
```suggestion
## Virtual desktop keyboard shortcuts
```

For better accessibility, only use H1s for the article title, and to make the "On this page" functionality working properly.